### ISSUE TITLE
In the Hasura Cloud, the processed-events of the Event item displays an error.

### ISSUE THREAD

**Issue by jieey1140** [hasura/graphql-engine#9651]
### Version Information

Server Version:
CLI Version (for CLI related issue):

### Environment
Cloud
 

### What is the current behaviour?
View Event Details in the Hasura Cloud

### What is the expected behaviour?
In Cloud Hasura, an error occurs when you click the detail view of any event in the Processed Events tab item on the Events tab.

### How to reproduce the issue?

1. Go to a project in the Hasura Cloud.
2. On the Events page, enter the Processed Events tab.
3. Press the View details button for any event.

### Screenshots or Screencast
<img width="546" alt="Screen Shot 2023-05-18 at 4 40 15 PM" src="https://github.com/hasura/graphql-engine/assets/85172908/43d38db2-4739-47b5-815f-13c925a88b43">

<img width="1145" alt="Screen Shot 2023-05-18 at 4 40 42 PM" src="https://github.com/hasura/graphql-engine/assets/85172908/229faac8-72a1-4b3a-ac3b-526c118f3c1d">

### Please provide any traces or logs that could help here.

```
r | "unexpected error while fetching event with id f28fea56-9e60-4ca2-b185-cc9e36004fe7: database query error"
-- | --
internal | Object { error: 'Failed to decode the following bytes in UTF-8: "\\NUL\\NUL\\NUL\\202"', prepared: true, statement: "\n SELECT *\n FROM hdb_catalog.event_invocation_logs\n WHERE event_id = $1\n ORDER BY created_at DESC LIMIT $2 OFFSET $3;\n ", … }
arguments | [ '(Oid 25,Just ("f28fea56-9e60-4ca2-b185-cc9e36004fe7",Binary))', '(Oid 20,Just ("\\NUL\\NUL\\NUL\\NUL\\NUL\\NUL\\NULd",Binary))', '(Oid 20,Just ("\\NUL\\NUL\\NUL\\NUL\\NUL\\NUL\\NUL\\NUL",Binary))' ]
error | 'Failed to decode the following bytes in UTF-8: "\\NUL\\NUL\\NUL\\202"'
prepared | true
statement | "\n SELECT *\n FROM hdb_catalog.event_invocation_logs\n WHERE event_id = $1\n ORDER BY created_at DESC LIMIT $2 OFFSET $3;\n "
pat

```

**Comment by manasag** [comment]
Hi @jieey1140, could you share the Hasura version you are using. There have been some improvements in recent versions around event fetching, and we don't see any issue as you mentioned, in the latest 2.25 version. Could you also try upgrading to this version if not already.

**Comment by Varun-Choudhary** [comment]
Hi @jieey1140, we are not able to reproduce this issue on an older version and the new version. It would be very helpful if you provide us with information about which version you are using, the minimal steps to reproduce this issue, metadata structure and the schema info. 
Thank you!

**Comment by jieey1140** [comment]
@manasag  @Varun-Choudhary  

Thank you for your reply. 
The current Cloud version is v2.25.0-cloud.2 and we are using a paid version.

**Comment by krushanbauva** [comment]
Hi @jieey1140 
Thank you for your reply.
We are able to reproduce the bug and have a fix for the same. It will be patched soon.

**Comment by jieey1140** [comment]
@krushanbauva  
Thank you for your confirmation. I hope this problem is resolved soon. 👍

**Comment by krushanbauva** [comment]
Hi @jieey1140
This has been patched in v2.25.1. Thank you for reporting it.
